lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -357,8 +361,12 @@ class MongoClientInterface {
                 filter: {
                     _id: objName,
                 },
+                // eslint-disable-next-line
                 update: {
                     _id: objName, value: objVal,
+                    'value.versionId': {
```

Comment:
this modifies the object pointed by the objVal reference. is it okay to do so? (seems also to be the reason for the eslint-disable comment

Reply:
The comment is just there because the linter does not like the quotes surrounding the dotted expression. Also if I don't quote the linter says 'unexpected token .'